java-checks/src/main/java/org/sonar/java/checks/TooLongLineCheck.java

```diff
@@ -99,10 +99,11 @@ private void visitFile() {
   }
 
```

Comment:
At line 93 we are calling `removeIgnoredPatterns(origLine)` for all lines, without checking if `origLine.length() > maximumLineLength`, I do not expect that a lot of lines exceed maximumLineLength, we should test this before calling `removeIgnoredPatterns(origLine)`.